testapi.pm

```diff
@@ -1119,19 +1120,23 @@ sub get_test_data {
 =head2 validate_script_output
 
   validate_script_output($script, $code [, timeout => $timeout] [,quiet => $quiet])
+  validate_script_output($script, $regexp [, timeout => $timeout] [,quiet => $quiet])
```

Comment:
Adding a second line with alternative syntax is uncommon for the test API and might be confusing. I suggest to specify the alternatives within a single line, e.g.

```perl
validate_script_output($script, $code | $regexp [, timeout => $timeout] [,quiet => $quiet])
```